code/iaas/model/src/main/java/io/cattle/platform/core/dao/impl/NetworkDaoImpl.java

```diff
@@ -1,30 +1,38 @@
 package io.cattle.platform.core.dao.impl;
 
 
-import static io.cattle.platform.core.model.tables.NetworkServiceProviderInstanceMapTable.*;
```

Comment:
boo... set your Eclipse settings to only require 1 static import to allow *.  (You don't need to change this now)